clang/include/clang/Basic/Linkage.h

```diff
@@ -20,7 +20,7 @@ namespace clang {
 
 /// Describes the different kinds of linkage
 /// (C++ [basic.link], C99 6.2.2) that an entity may have.
-enum Linkage : unsigned char {
+enum Linkage : unsigned {
```

Comment:
can you explain this change?

Reply:
This case is similar to `TypeDependence` we discuss in another thread, except that I can't find references to this type in data member declaration in our headers, which suggests that bit-field it stored in has hard-coded width, and has nothing to do with underlying type declared here.